src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx

```diff
@@ -29,6 +29,445 @@ Cumulative transaction data appears in APM on the [Transactions page](/docs/apm/
 * **Web**: Transactions are initiated with an HTTP request. For most organizations, these represent customer-centric interactions and thus are the most important transactions to monitor.
 * **Non-web**: [Non-web transactions](/docs/apm/transactions/intro-transactions/monitor-background-processes-other-non-web-transactions) are not initiated with a web request. They can include non-web worker processes, background processes, scripts, message queue activity, and other tasks.
 
+### Transaction sub-types
+
+Our agents have these transaction sub-types:
+
+<CollapserGroup>
+  <Collapser
+    id="agent-c-sdk"
+    title="C SDK"
+  >
+<table>
+  <thead>
+    <tr>
+      <td>
+        Sub-type
+      </td>
+
+      <td>
+        Description
+      </td>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        Custom
+      </td>
+
+      <td>
+        The execution of the named custom transaction. Usually recorded via manual instrumentation APIs. 
+      </td>
+    </tr>
+    
+    <tr>
+      <td>
+        Function
+      </td>
+
+      <td>
+        The invocation of the named function.
+      </td>
+    </tr>
+  </Collapser>
```

Comment:
I think we're missing a `</table>` here.

Reply:
Sorry, I had forgotten to commit.